src/utils/mdast-to-vdom.js

```diff
@@ -188,21 +187,14 @@ class VDOMTransformer {
     };
   }
 
-  /**
-   * Turns the MDAST into a basic DOM-like structure using Hyperscript
-   * @returns {Node} a simple DOM node (not all DOM functions exposed)
-   */
-  process() {
```

Comment:
The idea of this function is to be used in project-code, e.g. if you want to override the way HTML is being generated, you'd be able to register a new handler and then run `process` in your `pre.js`

Reply:
But I guess just using `getDocument` is easier.